recipes/momlevel/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "momlevel" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/momlevel-{{ version }}.tar.gz
+  sha256: e84390f2f588f221203e6837c11e8141f3d5396bb65644226906b041d4d6c5fc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
```

```suggestion
    - pip
    - setuptools-git-versioning
```
Ref: https://github.com/jkrasting/momlevel/blob/1cfa0f4c84172b77535dee87c9bb10c08424e2cb/pyproject.toml#L2